tools/extras/install_chainer.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Installs chainer with nn-gev dependencies
+# miniconda should be installed in $HOME/miniconda3/ 
+# Download cuDNN from "https://developer.nvidia.com/rdp/cudnn-download" and extract in "$HOME/cuda/"
+# and add their paths "export CFLAGS=-I$HOME/cuda/include" "export LDFLAGS=-L$HOME/cuda/lib64"
+
+cudnn_dir=$HOME/cuda
+cudnn_include_file=$cudnn_dir/include/cudnn.h
+cudnn_lib_dir=$cudnn_dir/lib64
+
+if [ ! -d $cudnn_lib_dir ] || [ ! -f $cudnn_include_file ]; then
```

Comment:
you should also check that the user has installed miniconda in the right place, and give appropriate instructions if they have not.